app/controllers/movies_controller.rb

```diff
@@ -4,6 +4,13 @@ class MoviesController < ApplicationController
 
   def index
     @movies = Movie.all
+    if params[:search] == ""
```

Comment:
not a great conditional, ie if they put in a bunch of spaces, perhaps consider using if params[:search].blank?